### ISSUE TITLE
Not working in iOS Hybrid app

### ISSUE THREAD

**Issue by ksivamuthu** [passy/angular-google-staticmaps#9]
Its not working in iOS when built with cordova.

**Comment by pdfowler** [comment]
I'm running into the same problem - it works fine when run against a local server, but once actually installed, it joes the broken image. I'm seeing this on both iOS and Android (ionic) cordova builds.

I've been playing around with my domain whitelist settings, but with no luck.

@ksivamuthu - have you found any workarounds or solutions to the problem?

**Comment by pdfowler** [comment]
Update: I was able to get this working by changing `BASE_URL` to use `https://...`, as opposed to the preferred `//...`. It seems that somewhere in the cordova access origin whitelist, it doesn't like to deal with protocol-less urls.

TLDR: 
Change line 13 in angular-google-staticmaps.js from:
`var BASE_URL = '//maps.googleapis.com/maps/api/staticmap?';`
to
`var BASE_URL = 'https://maps.googleapis.com/maps/api/staticmap?';`

PS: There may be more to this, I made other changes before landing on this. I will try to push a PR later today

**Comment by ksivamuthu** [comment]
Yes. I did the same and created the pull request. Its not accepted by the owner.

**Comment by ksivamuthu** [comment]
@passy Can you please consider my pull request again?

**Comment by passy** [comment]
@ksivamuthu I think I'd be okay with forcing `https`. I'm not married to the idea of having protocol-relative URLs. :)

Would you mind updating your PR and resubmitting it?

**Comment by pdfowler** [comment]
After spending a good chunk of time trying to handle it in the config, it seems that protocol-relative URL's have actually fallen out of favor:

http://www.paulirish.com/2010/the-protocol-relative-url/

Time to update _my_ code :)

**Comment by ksivamuthu** [comment]
@passy Updated the PR and resubmitted.
